sources/Debate/src/main/java/com/example/Debate/model/Activity.java

```diff
@@ -0,0 +1,15 @@
+package com.example.Debate.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Id;
+
+@Getter
```

Comment:
a @Data annotation could be used on model classess. https://projectlombok.org/features/Data